packages/core/src/dialects/abstract/query-generator.d.ts

```diff
@@ -93,6 +94,11 @@ export interface RemoveColumnQueryOptions {
   ifExists?: boolean;
 }
 
+export interface ShowAllTablesOptions extends QueryRawOptions {
+  // specify the schema to get the tables from
```

Comment:
```suggestion
  // specify the schema to get the tables from. Only supported by postgres
```

Reply:
I'm updating the dialects to support Schema.